src/components/visualizations/common/proptypes/DrillableItem.js

```diff
@@ -0,0 +1,9 @@
+// (C) 2007-2018 GoodData Corporation
+import PropTypes from 'prop-types';
+
+const DrillableItem = {
```

Comment:
PropTypes suffix ?

Reply:
What?

Reply:
as you can see all other propTypes definitions are using PropTypes suffix ;) in matter of consistency :)